asyncpg/connection.py

```diff
@@ -1110,7 +1122,8 @@ async def set_type_codec(self, typename, *,
         self._check_open()
 
         typeinfo = await self.fetchrow(
-            introspection.TYPE_BY_NAME, typename, schema)
+            introspection.TYPE_BY_NAME, typename, schema,
+            _disable_custom_codec=True)
```

Comment:
Let's not add private arguments to public methods.  Use `_execute` directly instead.